publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/grant/ContributorPermissionStrategy.java

```diff
@@ -14,25 +14,21 @@
 import no.unit.nva.model.role.RoleType;
 import no.unit.nva.publication.model.business.UserInstance;
 
-public class ContributorPermissionStrategy extends PermissionStrategy {
+public class ContributorPermissionStrategy extends GrantPermissionStrategy {
 
     public ContributorPermissionStrategy(Publication publication, UserInstance userInstance) {
         super(publication, userInstance);
     }
 
     @Override
-    protected boolean allowsAction(PublicationOperation permission) {
+    public boolean allowsAction(PublicationOperation permission) {
```

Comment:
can we avoid public here? Its very easy that ContributorPermissionStrategy is misused by direct access (already happened)